examples/minimal_agent_production/steps/analyze.py

```diff
@@ -48,102 +44,125 @@ def perform_llm_analysis(
         - latency_ms: Processing time in milliseconds
 
     Raises:
-        ImportError: If litellm is not available
+        ImportError: If openai is not available
     """
-    logger.info(f"Starting LLM analysis for document: {filename}")
-
     try:
-        # Lazy imports to avoid hard dependency when offline
-        import instructor
-        from litellm import completion
+        # Lazy import to avoid hard dependency when offline
+        import json
+
+        from openai import OpenAI
 
         # Clean content for better LLM analysis
         cleaned_content = clean_text_content(content)
         content_preview = cleaned_content[:2000]
 
-        # Construct analysis prompt
-        prompt = f"""Analyze this document and provide structured analysis.
+        # Standard analysis request
+        analysis_request = """
+Respond with a JSON object containing:
+- summary: A concise 2-3 sentence summary
+- keywords: Array of 5 relevant keywords
+- sentiment: One of "positive", "negative", or "neutral"
+- readability: One of "easy", "medium", or "hard"
+"""
```

Comment:
These prompts IMO should all be extracted out somewhere separate. Constant or
even a prompts.py file.